qt/widgets/plotting/inc/MantidQtWidgets/Plotting/Mpl/ExternalPlotter.h

```diff
@@ -6,30 +6,39 @@
 // SPDX - License - Identifier: GPL - 3.0 +
```

Comment:
Can I ask why this was moved to `mplcpp`? This library was intended to solely be a C++ wrapper around matplotlib functionality without adding any mantid specifics like workspaces. The `plotting` library, probably best renamed `WorkspacePlotting` thinking about it, was  intended to be the layer that then mixed mplcpp and workspaces to define interfaces that could be used without it's users linking directly to matplotlib.

Reply:
That might be my fault for suggesting this location to Matt. This file already appears to be in the `qt/widgets/plotting` directory rather than `qt/widgets/mplcpp` -  is this still the wrong location to be in the `plotting` library?

Reply:
Nope you're correct that `qt/widgets/plotting` is the correct location for the plotting library. I'll agree the name of the plotting library is overly generic though and maybe I need to consider putting a README in each library to describe its intent.